app/routes.js

```diff
@@ -1,10 +1,12 @@
-// const controller = require('./controllers/controller');
 const { healthCheck } = require('./controllers/healthCheck');
+const validateSchema = require('./middlewares/validateSchema');
 const usersController = require('./controllers/users');
+const { userSchema, loginSchema } = require('./validations');
 const checkIfEmailExists = require('./middlewares/emailValidation');
 
 exports.init = app => {
   app.get('/health', healthCheck);
 
-  app.post('/users', [checkIfEmailExists], usersController.signUp);
+  app.post('/users', [validateSchema(userSchema)], [checkIfEmailExists], usersController.signUp);
```

Comment:
You can do: `app.post('/users', [validateSchema(userSchema), checkIfEmailExists], usersController.signUp);`